ReactNativeClient/lib/components/select-date-time-dialog.js

```diff
@@ -22,7 +22,7 @@ class SelectDateTimeDialog extends React.PureComponent {
 			this.setState({ date: newProps.date });
 		}
 
-		if ('shown' in newProps) {
+		if ('shown' in newProps && newProps.shown !== this.shown_) {
```

Comment:
Is this fix related to the share extension? Seems to be an unrelated bug fix.

Reply:
Yes, it's not directly related.

It's just that with this change it started to occur every time (when sharing). And I spent a lot of time before realizing that it's been there all along and it's not that big of a deal. Though the warning does make it seem scary with the mention of memory leak.

But since I've fixed it anyway figured might as well include it (I'm a fan of the boy scout rule),